examples/createCollection.ts

```diff
@@ -0,0 +1,39 @@
+import { AlchemyProvider } from '@ethersproject/providers';
+import { Wallet } from '@ethersproject/wallet';
+import { ImmutableX, Config, CreateCollectionRequest } from '@imtbl/core-sdk';
+import { requireEnvironmentVariable } from './libs/utils';
+
+(async () => {
+  const privateKey = requireEnvironmentVariable('PRIVATE_KEY');
+  const alchemyKey = requireEnvironmentVariable('ALCHEMY_API_KEY');
+  const wallet = new Wallet(privateKey);
+  const provider = new AlchemyProvider('goerli', alchemyKey);
+  const signer = wallet.connect(provider);
+
+  const imxClient = new ImmutableX(Config.SANDBOX);
+
+  const createCollectionParams: CreateCollectionRequest = {
+    contract_address: '',
+    name: '',
+    owner_public_key: wallet.publicKey,
+    project_id: 0,
+  };
+
+  try {
+    const createCollectionResponse = await imxClient.createCollection(
+      signer,
+      createCollectionParams,
+    );
+
+    console.log('createCollection', JSON.stringify(createCollectionResponse));
```

Comment:
Lot of inconsistencies with these logs. Can we stick to one?

```suggestion
    console.log('createCollectionResponse', JSON.stringify(createCollectionResponse));
```

Reply:
[a7f99be](https://github.com/immutable/imx-core-sdk/pull/195/commits/a7f99bee29bf808011d4355d4e30590187b35fcc)